src/World.cpp

```diff
@@ -64,313 +64,301 @@ World::~World() {
-        playerColl.w = PlayerWidth;
-        playerColl.y = Y;
-        playerColl.h = PlayerHeight;
-        playerColl.z = Z - PlayerLength / 2.0;
-        playerColl.l = PlayerLength;
-
-        const Section &section = this->GetSection(it);
-        for (int x = 0; x < 16; x++) {
-            for (int y = 0; y < 16; y++) {
-                for (int z = 0; z < 16; z++) {
-                    BlockId block = section.GetBlockId(Vector(x, y, z));
-                    if (block.id == 0 || block.id == 31 || block.id == 37 || block.id == 38 || block.id == 175)
-                        continue;
-                    AABB blockColl{ (x + it.x * 16.0),
-                        (y + it.y * 16.0),
-                        (z + it.z * 16.0), 1, 1, 1 };
-                    if (TestCollision(playerColl, blockColl))
-                        return true;
-                }
-            }
-        }
-    }
-    return false;
-}
-
 std::vector<Vector> World::GetSectionsList() {
-    sectionsListMutex.lock();
-    auto vec = sectionsList;
-    sectionsListMutex.unlock();
-    return vec;
+	sectionsListMutex.lock();
+	auto vec = sectionsList;
+	sectionsListMutex.unlock();
+	return vec;
 }
 
 static Section fallbackSection;
 
 const Section &World::GetSection(Vector sectionPos) {
-    auto result = sections.find(sectionPos);
-    if (result == sections.end()) {
-         //LOG(ERROR) << "Accessed not loaded section " << sectionPos;
-        return fallbackSection;
-    }
-    else {
-        return *result->second.get();
-    }
+	auto result = sections.find(sectionPos);
+	if (result == sections.end()) {
+		 //LOG(ERROR) << "Accessed not loaded section " << sectionPos;
+		return fallbackSection;
+	}
+	else {
+		return *result->second.get();
+	}
 }
 
 // TODO: skip liquid blocks
 RaycastResult World::Raycast(glm::vec3 position, glm::vec3 direction) {
-    const float maxLen = 5.0;
-    const float step = 0.01;
+	const float maxLen = 5.0;
+	const float step = 0.01;
 	glm::vec3 pos = glm::vec3(0.0);
-    float len = 0;
-    Vector blockPos = Vector(position.x,position.y,position.z);
-    while (GetBlockId(blockPos) == BlockId{0, 0} && len <= maxLen) {
-        pos = position + direction * len;
-        len += step;
-        blockPos = Vector(floor(pos.x), floor(pos.y), floor(pos.z));
-    }
-
-    RaycastResult result;
-    result.isHit = !(GetBlockId(blockPos) == BlockId{0, 0});
-    result.hitPos = VectorF(pos.x,pos.y,pos.z);
-    result.hitBlock = blockPos;
-    return result;
-}
+	float len = 0;
+	Vector blockPos = Vector(position.x,position.y,position.z);
+	while (GetBlockId(blockPos) == BlockId{0, 0} && len <= maxLen) {
+		pos = position + direction * len;
+		len += step;
+		blockPos = Vector(floor(pos.x), floor(pos.y), floor(pos.z));
+	}
 
+	RaycastResult result;
+	result.isHit = !(GetBlockId(blockPos) == BlockId{0, 0});
+	result.hitPos = VectorF(pos.x,pos.y,pos.z);
+	result.hitBlock = blockPos;
+	return result;
+}
+inline bool ISUNCOLLIDEBLE(int id) {
```

Comment:
That's not what I mean... Use CamelCase - for functions.
```suggestion
inline bool IsUncollideble(int id) {
```

Reply:
And use `BlockId id` insted of `int id`